components/templates/home/HomeTemplate.tsx

```diff
@@ -10,12 +10,15 @@ import { HeroSection } from './_hero_section';
 import { PricingSection } from './_pricing_section';
 
 export const HomeTemplate: TopLevelTemplate = () => {
-  const { translation } = useContext(TranslationContext);
+  const { locale, translation } = useContext(TranslationContext);
+  useEffect(() => {
+    document.documentElement.lang = locale;
```

Comment:
Should this be added to all of the other templates too?

Reply:
This should be on the login one, will fix.

Reply:
Sounds good
Try adding to TranslationWrapper